t/plugin/proxy-rewrite3.t

```diff
@@ -313,6 +313,57 @@ ngx.var.request_uri: /print_uri_detailed
                 }]]
                 )
 
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.say(body)
+        }
+    }
+--- response_body
+passed
+
+
+
+=== TEST 13: rewrite X-Forwarded-Host
```

Comment:
This test isn't relative to this feature?

Reply:
Nope, but this change is to remove duplicate sections from your advice.

<img width="575" alt="image" src="https://user-images.githubusercontent.com/11229089/203016510-11a244cd-580b-4871-8906-0f4e0fc1e537.png">